### DISCUSSION TITLE
Passing Object Args

### DISCUSSION THREAD

**Discussion by 3dylson** [Tlaster/PreCompose#199] [Q&A] [answered]
How do I pass an Object other then String as an argument?

**Comment by Tlaster** [comment, marked as answer]
You can checkout [this stackoverflow](https://stackoverflow.com/questions/67121433/how-to-pass-object-in-navigation-in-jetpack-compose), or better, if you follow the [Guide to app architecture](https://developer.android.com/topic/architecture), you don't need to passing object args.

**Reply by 3dylson** [reply]
Thks!
